ChakraCore Samples/CrossPlatform/README.md

```diff
@@ -0,0 +1,28 @@
+### Embedding ChakraCore on Linux and macOS
```

Comment:
Given there's no OS X build guide of ChakraCore, let's either hold off the OS X part or get the build guide on ChakraCore wiki first just like the [Linux guide](https://github.com/Microsoft/ChakraCore/wiki/Building-ChakraCore#linux).